app/views/directives/ui-ace-yaml.html

```diff
@@ -0,0 +1,44 @@
+<ng-form name="$ctrl.form">
+  <div class="form-group" id="yaml-file">
+    <osc-file-input
+      ng-if="$ctrl.showFileInput"
+      model="$ctrl.fileUpload"
+      drop-zone-id="yaml-file"
+      help-text="Upload file by dragging & dropping, selecting it, or pasting from the clipboard."
+      ng-disabled="false"
+      hide-clear="true"></osc-file-input>
+
+    <div class="edit-yaml-errors">
+      <!--
+        Since the YAML parser quits after the first error, don't give a count.
+        Just show an indicator and link to the first problem line.
+      -->
+      <div ng-if="firstError = $ctrl.annotations.error[0]">
```

Comment:
It will move the cursor to the right line, so it's still useful even if the error was visible